tests/tracking/test_utils.py

```diff
@@ -168,6 +169,62 @@ def test_get_store_databricks_profile():
         assert 'mycoolprofile' in str(e_info.value)
 
 
+def test_standard_store_registry():
+    mock_entrypoint = mock.Mock()
+    mock_entrypoint.name = "mock-scheme"
+
+    with mock.patch(
+        "entrypoints.get_group_all", return_value=[mock_entrypoint]
+    ):
+        # Entrypoints are registered at import time, so we need to reload the
+        # module to register the entrypoint given by the mocked
+        # extrypoints.get_group_all
+        reload(mlflow.tracking.utils)
+
+        expected_standard_registry = {
+            '',
+            'file',
+            'http',
+            'https',
+            'postgresql',
+            'mysql',
+            'sqlite',
+            'mssql',
+            'databricks',
+            'mock-scheme'
+        }
+        assert expected_standard_registry.issubset(
+            mlflow.tracking.utils._tracking_store_registry._registry.keys()
+        )
+
+
+def test_plugin_registration():
+    tracking_store = TrackingStoreRegistry()
+
+    mock_plugin = mock.Mock()
+    tracking_store.register("mock-scheme", mock_plugin)
+    assert "mock-scheme" in tracking_store._registry
+    assert tracking_store.get_store("mock-scheme://fake-host/fake-path") == mock_plugin.return_value
```

Comment:
May be good to have a test that verifies we invoke the plugin with the appropriate arguments (currently, just store_uri and artifact_uri).